lib/ansible/cli/galaxy.py

```diff
@@ -526,8 +549,12 @@ def add_install_options(self, parser, parents=None):
                                              'This does not apply to collections in remote Git repositories or URLs to remote tarballs.'
                                         )
         else:
-            install_parser.add_argument('-r', '--role-file', dest='requirements',
-                                        help='A file containing a list of roles to be installed.')
+            if self._implicit_role:
+                install_parser.add_argument('-r', '--requirements-file', '--role-file', dest='requirements',
```

Comment:
Should we add a changelog entry mentioning that `--requirements-file` was added?

Reply:
Yeah... though I added this only to make the description for the positional arg match between `ansible-galaxy collection install --help` and `ansible-galaxy install --help`. Maybe it would be better for me to just to use the abbreviation `-r` in the --help description instead for simplicity. I don't imagine anyone really wants to type `--requirements-file`.

Reply:
Sounds good. Though in https://github.com/ansible/ansible/pull/83919/commits/fe4e5f7f2be04ce430a9b91651105f9330e18479 I think we lost the mention of possibility that  collections can be specified in a file passed via `-r`, no?

Reply:
Fixed.